support/main/default/classes/TriggerControlTest.cls

```diff
@@ -1,137 +1,243 @@
+        Boolean contactHandler1Fired = true;
+        runTest(acctHandler1Fired, acctHandler2Fired, contactHandler1Fired);
+    }
+
+    @isTest
+    private static void testContactDisable() {
+        Map<String, TriggerConfigService.ObjectConfig> mdt = buildMetadata();
+        mdt = disableObj(mdt, 'contact');
+        TriggerConfigService.configMetadata = mdt;
+        Boolean acctHandler1Fired = true;
+        Boolean acctHandler2Fired = true;
+        Boolean contactHandler1Fired = false;
+        runTest(acctHandler1Fired, acctHandler2Fired, contactHandler1Fired);
+    }
+
+    @isTest
+    private static void testAccountDisable() {
+        Map<String, TriggerConfigService.ObjectConfig> mdt = buildMetadata();
+        mdt = disableObj(mdt, 'account');
+        TriggerConfigService.configMetadata = mdt;
+        Boolean acctHandler1Fired = false;
+        Boolean acctHandler2Fired = false;
+        Boolean contactHandler1Fired = true;
+        runTest(acctHandler1Fired, acctHandler2Fired, contactHandler1Fired);
+    }
+
+    @isTest
+    private static void runAllEvents() {
+        Map<String, TriggerConfigService.ObjectConfig> mdt = new Map<String, TriggerConfigService.ObjectConfig>();
+        TriggerConfigService.ObjectConfig acct = generateObjTestData('account', true);
+        TriggerConfigService.HandlerConfig h3 = new TriggerConfigService.HandlerConfig();
+        h3.className = 'handler3';
+        h3.enabled = true;
+        h3.order = 0;
+        h3.autoBind = true;
+        h3.beforeInsert = true;
+        h3.afterInsert = true;
+        h3.beforeUpdate = true;
+        h3.afterUpdate = true;
+        h3.beforeDelete = true;
+        h3.afterDelete = true;
+        h3.afterUndelete = true;
+        acct = addChildHandlerToObj(acct, h3);
+        mdt.put('account', acct);
+        TriggerConfigService.configMetadata = mdt;
+        Id aId = createAccount();
+        Account a = new Account(Id = aId);
+
+        update a;
+        delete a;
+        undelete a;
+
+        // all events should have fired:
+        System.assert(Handler3.events.contains('beforeInsert'), 'beforeInsert should have fired');
+        System.assert(Handler3.events.contains('afterInsert'), 'afterInsert should have fired');
+        System.assert(Handler3.events.contains('beforeUpdate'), 'beforeUpdate should have fired');
+        System.assert(Handler3.events.contains('afterUpdate'), 'afterUpdate should have fired');
+        System.assert(Handler3.events.contains('beforeDelete'), 'beforeDelete should have fired');
+        System.assert(Handler3.events.contains('afterDelete'), 'afterDelete should have fired');
+        System.assert(Handler3.events.contains('afterUndelete'), 'afterUndelete should have fired');
+        System.assert(Handler3.events.size() == 7, 'All events should have fired one time');
+    }
+
+    // not sure how to implement a test on legacy binding methods without putting legacy binding in the trigger
```

Comment:
that's what the first 5 tests already do, since the trigger is in the support directory it doesn't get packaged

this test case is covered, can delete comment